src/pacer.js

```diff
@@ -197,6 +208,7 @@ let PACER = {
     let bigFile = document.getElementById('file_too_big')
     let buttonText = inputs.length ? inputs[inputs.length - 1].value.includes('Download') : false
     let mainContent = document.getElementById("cmecfMainContent");
+    if (!mainContent){ return false }
```

Comment:
When does this happen? What caused you to add this?

Reply:
The `cmecfMainContent` is the parent `div` that encloses almost all the `forms` in **CM/ECF**, but that `div `doesn't exist in the PACER website so the **extension** always gets `null ` while the user is using one of the pages (like Manage my account) in websites like https://pacer.psc.uscourts.gov/.

if the value of mainContent is null then the extension will find an Uncaught typeError in line 227 of that file when it tries to get the `lastChild` of the div. I added the validation before it reaches the possible typeError,